src/OpenIDConnectClient.php

```diff
@@ -698,7 +698,11 @@ public function refreshToken($refresh_token) {
         $token_params = http_build_query($token_params, null, '&');
 
         $json = json_decode($this->fetchURL($token_endpoint, $token_params));
-        $this->refreshToken = $json->refresh_token;
+        $this->accessToken = $json->access_token;
```

Comment:
in case the request is invalid (returns with an error invalid_grant), this will fail as there is no access token in the context